daemon/meson.build

```diff
@@ -13,14 +21,19 @@ if get_option('enable-machine-learning-agent')
               '--generate-c-code', 'pipeline-dbus',
               '--output-directory', meson.current_build_dir(),
               '@INPUT@'])
-
   nns_ml_agent_gen_srcs += gdbus_gen_src
-
-  nns_ml_agent_srcs += join_paths('main.c')
-  nns_ml_agent_srcs += join_paths('modules.c')
-  nns_ml_agent_srcs += join_paths('gdbus-util.c')
   nns_ml_agent_srcs += join_paths('pipeline-module.c')
```

Comment:
move this to line 14?

Reply:
Applied as below. Thank you for notifying me.
```c++
  nns_ml_agent_srcs += join_paths('gdbus-util.c')
  nns_ml_agent_srcs += join_paths('service-db.cc')
  nns_ml_agent_srcs += join_paths('pipeline-module.c')
  nns_ml_agent_srcs += join_paths('model-dbus-impl.cc')
```